### DISCUSSION TITLE
ffmpg question

### DISCUSSION THREAD

**Discussion by Mike92115** [samemory/homebridge-eufy-security-control#49] [Q&A]
Hello,

I am not getting any snapshots or streaming, so I reinstalled /homebridge/node_modules/ffmpeg-for-homebridge/ffmpeg

but I am seeing 2 warning, and wanted to ask if everyone else got these or if they could be the reason that I am not seeing snapshots or steaming?

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/eufy-node-client/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

Thank you,
Mike

**Comment by Mike92115** [comment]
Update, I removed the plugin, and made sure the items were removed from HomeKit and re-installed it is working now.  But I still had the warning above for FFMPG.

**Reply by lenoxys** [reply]
hello this is warning message, that's shouldn't block you to use this plugin

**Reply by Mike92115** [reply]
Thank you for letting me know, just wanted to check.

Mike
